src/docs/previews/tree/main/tailwind.svelte

```diff
@@ -0,0 +1,102 @@
+<!-- https://www.w3.org/WAI/ARIA/apg/patterns/treeview/ -->
+
+<script lang="ts">
+	import { setContext } from 'svelte';
+	import { createTreeViewBuilder } from '@melt-ui/svelte';
+
+	import Tree from './tree.svelte';
+	import type { TreeItem } from './tree.svelte';
+	import { generateId } from '@melt-ui/svelte/internal/helpers';
+
+	const {
+		tree,
+		label,
+		item,
+		group,
+		currentFocusedItem,
+		currentSelectedItem,
```

Comment:
@TGlide I'll update them. Do you think that `focusedItem` and `selectedItem` should contain the node, or should we have two separate stores, one which holds the focused value and one which holds the focused ID. Since in a tree you can have multiple branches which might have the same value it would be important to also return the ID. Or should we just leave it like it currently is?

Reply:
I think having the node is sufficient!